doc/sphinx/addendum/parallel-proof-processing.rst

```diff
@@ -154,10 +154,10 @@ to a worker process. The threshold can be configured with
 Batch mode
 ---------------
 
-When |Coq| is used as a batch compiler by running `coqc` or `coqtop`
--compile, it produces a `.vo` file for each `.v` file. A `.vo` file contains,
-among other things, theorem statements and proofs. Hence to produce a
-.vo |Coq| need to process all the proofs of the `.v` file.
+When |Coq| is used as a batch compiler by running `coqc`, it produces
+a `.vo` file for each `.v` file. A `.vo` file contains, among other
+things, theorem statements and proofs. Hence to produce a .vo |Coq|
+need to process all the proofs of the `.v` file.
```

Comment:
Use double backticks throughout this paragraph. Same thing in the other chapter.

Reply:
Thanks @zimmi48, updated.